api-contracts/openapi/components/schemas/v1/task.yaml

```diff
@@ -393,7 +397,7 @@ V1TaskTiming:
     - taskId
     - taskInsertedAt
     - tenantId
-
+    - workflowRunId
```

Comment:
I think making this required will be breaking, since if we generate the Python SDK pydantic will treat this as `str` and not `str | None`, and then will fail on older APIs (I've made this mistake at least once 😭 )

Reply:
I don't think so since the openapi generator generates Pydantic, which I think is actually a good thing at the end of the day but is annoying for things like this